integration/localnet/bootstrap.go

```diff
@@ -313,6 +315,7 @@ func prepareService(container testnet.ContainerConfig, i int) Service {
 			fmt.Sprintf("--tracer-enabled=%t", true),
 			"--profiler-dir=/profiler",
 			"--profiler-interval=2m",
+			fmt.Sprintf("--admin-addr=:%v", 9002+n),
```

Comment:
Is the `+n` so that the `admin-addr` is not unique and does not collide on e.g. localnet? If so, I noticed on localnet that the `--ingress-addr=collection_2:9000` option uses a fixed port but appears to use a custom network interface to differentiate. Would that be an alternative option to using `+n`? Pros? Cons?

Reply:
Reverted, this will be a fixed internal port